.github/workflows/worker.yml

```diff
@@ -19,6 +19,13 @@ on:
         required: false
         type: string
         description: "Prefix for the artifact name"
+      artifacts_to_download:
+        required: false
+        type: string
+        description: "Artifacts to download. For example: 'unit-test-reports-*'"
+
+env:
+  FORCE_COLOR: 2
```

Comment:
what does this mean?
why its being set here?

Reply:
The output of jacoco that showed whether a line is covered or not --- it doesn't show up in Green/Red without this.

Reply:
Although I guess this could be moved to the jacoco sh file 🤔 
Will try that out

Reply:
Moved FORCE_COLOR to the jacoco script. Works.